vitest.workspace.ts

```diff
@@ -0,0 +1,52 @@
+import { defineWorkspace } from 'vitest/config';
+import react from '@vitejs/plugin-react';
+
+const packages = ['charts', 'date-pickers'];
```

Comment:
Maybe to keep it simpler, e.g. if we have a none x-prefix package.

```suggestion
const packages = ['x-charts', 'x-date-pickers'];
```